key-derivation.md

```diff
@@ -72,13 +68,26 @@ credentialMac = SHA256HMAC(seedKey XOR credentialMacMask, rpId || version || uni
 
 Derive the private key **`es256SPrivateKey`**  from the `seedKey` on the authenticator, the `rpId` from the relying party, and the `credentialMac` field from the `credentialId` (which, as a MAC, effectively encapsulates the three other fields of `credentialId`.
 
+
+Following [NIST FIPS 186.4](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf) Section Section B.4.2,
+derive the secret key by testing randomly-generated 32-byte candidate sequences c_0, c_1, ... c_i of the form 
```

Comment:
I agree with this "reference to authority", it's what is meant by "rejection sampling". We only differ in how to generate the candidate sequence.